### ISSUE TITLE
Menu and Navigation Enhancement Possible?

### ISSUE THREAD

**Issue by f00d4tehg0dz** [boxeehacks/boxeehack#306]
To the Dev(s),
I absolutely love all the work thats been put into this so far. 
I do have a huge and troublesome question...

I currently average around 1000 movies, and 100 Full TV Shows.
And the load times for the Movies Menu, and Tv Show Menu is downright abysmal. 
As well as navigation around inside those Menu's.
Is there anyway to fix this issue?
The only solution I've ever found was to clear my cache/media files every few weeks...

**Comment by terosteros** [comment]
I am finding the same thing it only gets worse as well with 2100 movies and 510 TV Shows. Maybe it's just the trying to cache so many images across a network etc.

**Comment by f00d4tehg0dz** [comment]
If thats the case then...
I propose we get the ability to have the images store on the boxee box itself!!!!
Something to reduce all this lag.

_sidenote_ I do not envy you with all those files haha. Gotta be horrendous.

**Comment by terosteros** [comment]
Yeah i don't know if that would be possible would need to be dev input i don't know if they can though as i think would require access to the box. But attaching a 64 or 128gb flash drive and using that for cache would be the easiest or even if it copies the images from the network...

**Comment by boxeehacks** [comment]
All the poster images are already stored on the box itself (that's that media cache). But it gets stored at quite high resolution, which is using a lot of memory, disk and performance in the process. I'm looking at ways to process that cache and resizing stuff to lower res for the posters (since they don't need such high res). But that's something for the long-term roadmap (will be a lot of work).

What you can do now is resize all your posters yourself to lower resolution. This will speed up those menus a lot.

**Comment by KarmaMole** [comment]
How can we resize the posters ourselves? They don't seem to reside on the external usb?
